### DISCUSSION TITLE
Incorrect space key position in KeyCounterDisplay in osu!mania

### DISCUSSION THREAD

**Discussion by ZeroAurora** [ppy/osu#25944] [Q&A] [answered]
KeyCounterDisplay displays key misplaced, as shown in below screenshot.
Keymap is the default sdf[space]jkl. Obviously B2 is the space key, but is placed in front of other keys.
I'm not certain if it's intended (since keymap can be changed), but it's surely counterintuitive. Allowing the customization of how the keys are placed in skin can be helpful.
![图片](https://github.com/ppy/osu/assets/25359455/80b10e62-a515-410d-a78d-9754e054a88c)

Not posting logs here (not a bug).

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/26589
